docs/CONFIG-VARS.md

```diff
@@ -54,22 +54,26 @@ For recommendations on how to set these variables in your environment, see [Auth
 
 ## Admin Access
 
-By default, the API of the Azure resources that are being created is only accessible through authenticated Azure clients (such as the Azure Portal,
-the `az` CLI, the Azure Shell, etc.).
-To allow access for other administrative client applications (for example `kubectl`, `psql`, etc.), you must open the Azure firewall to allow access from your source
-IP addresses.
-To do this, specify ranges of IP addresses in [CIDR notation](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing).
-Contact your Network Administrator to find the public CIDR range of your network.
+By default, the public endpoints of the Azure resources that are being created
+are only accessible through authenticated Azure clients
+(such as the Azure Portal, the `az` CLI, the Azure Shell, etc.).
+To allow access for other administrative client applications (for example `kubectl`, `psql`, `ssh` etc.), you can set Network Security Group (NSG) rules to control access from your source IP addresses.
+
+To do set these permissions as part of this Terraform script, specify ranges of IP addresses in [CIDR notation](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) with the following variables.
+
+NOTE: When deploying infrastructure into a private network (e.g. a VPN), with no public endpoints, the options documented in this block are not applicable.
+
+NOTE: The script will either create a new NSG, or use an existing NSG, if specified in the [`nsg_name`](#use-existing) variable. 
```

Comment:
Again, no Azure Firewall, this has been replaced here. Correct?

Reply:
again, Azure Firewall was never in play here.